index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bierproductie: Info Pages</title>
+    <link rel="stylesheet" href="styles/main.css"/>
+    <script src="scripts/script.js" defer></script>
+    <script src="scripts/maintain.js" defer></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  </head>
+  <body>
+
+    <!-- Side navigation -->
+    <div class="sidenav">
+      <span class="iconify marked-page" onclick="location.href='/';" data-icon="cil:touch-app" data-inline="false">
+      </span>
+      <span class="iconify" onclick="location.href='/settings.html';" data-icon="carbon:settings" data-inline="false">
+      </span>
+    </div>
+
+    <!-- topbar -->
+    <header>       
+      <h2>Bierproductie</h2>
```

Comment:
Would be good if "Bierproductie" were aligned with some of the other elements around it. It kinda floats on its own as it is now. 

![image](https://user-images.githubusercontent.com/55458479/99436992-1b5a8780-2912-11eb-9696-db8e6046ade4.png)

Maybe move it back so it aligns with the outer grey box.

Reply:
Like this in the mockup? 
![image](https://user-images.githubusercontent.com/55736468/99453747-b741be80-2925-11eb-8c1c-4dcbd083d959.png)

Reply:
Ohh I can see there should not be anyborder on the highligted one

Reply:
I just thought about if the title could not line up with the body content, because it depens how big your screen is. 
Is this fine ?
![image](https://user-images.githubusercontent.com/55736468/99513593-1980da80-298b-11eb-842a-de53b2435318.png)
It lines up with the menu on the left?

Reply:
We can have it start all on the left?

Reply:
Ahh, of course, didn't think about that. 


> 
> 
> I just thought about if the title could not line up with the body content, because it depens how big your screen is.
> Is this fine ?
> ![image](https://user-images.githubusercontent.com/55736468/99513593-1980da80-298b-11eb-842a-de53b2435318.png)
> It lines up with the menu on the left?

This is fine yes.